app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -74,9 +74,17 @@ function DocumentCaptureReviewIssues({
           ]}
         />
       )}
-      <DocumentFrontAndBackCapture defaultSideProps={defaultSideProps} value={value} />
+      <DocumentFrontAndBackCapture
+        defaultSideProps={defaultSideProps}
+        value={value}
+        selfieCaptureEnabled={selfieCaptureEnabled}
+      />
       {selfieCaptureEnabled && (
-        <SelfieCaptureWithHeader defaultSideProps={defaultSideProps} selfieValue={value.selfie} />
+        <SelfieCaptureWithHeader
+          defaultSideProps={defaultSideProps}
+          selfieValue={value.selfie}
+          selfieCaptureEnabled={selfieCaptureEnabled}
```

Comment:
Isnt' this already true when the component get rendered?

Reply:
Do you mean, if we render this component, does that mean `selfieCaptureEnabled` must be true?

If so, that is a good point. Since both `SelfieCaptureWithHeader` and `DocumentFrontAndBackCapture` render `DocumentSideAcuantCapture`, I was trying to keep the logic and interfaces the same. Maybe that's not necessary though.

I will say too that I discussed this with Eileen today and may timebox approaching this work in a different way, to avoid needing to pass down this data through so many components.

Reply:
@night-jellyfish yes that's what I meant, unless the component is used other places where it can be false, Also to avoid property drill down, one way is through react context.

Reply:
+1, working with React context was the approach Brittany and I discussed yesterday.  I agree that it could work well in this scenario. 👍🏻

Reply:
Yes, exactly. The prep for that work is up now: https://github.com/18F/identity-idp/pull/10183.